### ISSUE TITLE
twisted.internet.defer.__all__ excludes "returnValue"

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#3685]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#3685|
|Type|enhancement|
|Created|2009-03-11 19:55:08Z|
|Branch|https://github.com/twisted/twisted/tree/expose-returnvalue-3685|

`returnValue` is part of the public API; it's necessary for many uses of `inlineCallbacks`; it should be part of `__all__`.

<details><summary>Searchable metadata</summary>

```
trac-id__3685 3685
type__enhancement enhancement
reporter__exarkun exarkun
priority__highest highest
milestone__ 
branch__branches_expose_returnvalue_3685 branches/expose-returnvalue-3685
branch_author__exarkun exarkun
status__closed closed
resolution__fixed fixed
component__core core
keywords__easy easy
time__1236801308000000 1236801308000000
changetime__1237567163000000 1237567163000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_1"></a>|@exarkun commented|
|-|-|

(In [26375]) Branching to 'expose-returnvalue-3685'

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun commented|
|-|-|

(In [26376]) Add returnValue to `__all__`

refs [#3685](https://github.com/twisted/twisted/issues/3685)

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun removed owner|
|-|-|

[Build results](http://buildbot.twistedmatrix.com/boxes-supported?branch=/branches/expose-returnvalue-3685)

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_4"></a>|@glyph set owner to @exarkun|
|-|-|

Merge.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_5"></a>|@exarkun set status to `closed`|
|-|-|

(In [26480]) Merge expose-returnvalue-3685

Author: exarkun
Reviewer: glyph
Fixes: [#3685](https://github.com/twisted/twisted/issues/3685)

Add `returnValue` to `twisted.internet.defer.__all__` since it is certainly part
of the public API of that module.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
